clients/python-wrapper/lakefs/branch.py

```diff
@@ -124,6 +124,25 @@ def get_commit(self):
         self._commit = None
         return super().get_commit()
 
+    def cherry_pick(self, reference: ReferenceType, parent_number: Optional[int] = None) -> Commit:
+        """
+        Cherry-pick a given reference onto the branch.
+
+        :param reference: ID of the reference to cherry-pick.
+        :param parent_number: When cherry-picking a merge commit,
+            the parent number (starting from 1) relative to which to perform the operation.
+            The destination branch is parent 1, which is the default behaviour.
```

Comment:
Not sure what this means.  Either I need more coffee or
```suggestion
            The default branch is parent 1, which is the default behaviour.
```
(or both).

Reply:
This comes straight from the docstring of `lakefs_sdk.CherryPickCreation` fwiw.

Reply:
@nicholasjng I tend to agree with @arielshaqed about this one and give you permission to modify the description in our swagger.yaml
I suggest the following:

> When cherry-picking a merge commit, the parent number (starting from 1) with which to perform the diff.
The default branch is parent 1.

Reply:
# 😳